### PR TITLE
p5-webservice-musicbrainz: new port, version 1.0.4

### PR DESCRIPTION
**Pull request by chrstphrchvz**
#### Description

(See new [comment](https://github.com/macports/macports-ports/pull/1973#issuecomment-395864348) below)
~~There are 2 new commits:~~
~~- add port p5-webservice-musicbrainz~~
~~- abcde: re-enable abcde-musicbrainz-tool (all of its other Perl dependencies are already available as ports)~~

~~Note: this depends on https://github.com/macports/macports-ports/pull/1967 (abcde 2.8.1 is incompatible with WebService::MusicBrainz >0.94). I have included that PR's commit for CI testing to work, under the assumption that one will get merged before this one. If that PR should be considered superseded by this one, I can squash the changes to abcde.~~

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.13.5 17F77
Xcode 9.4 9F1027a

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] ~~referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?~~

- [x] checked your Portfile with `port lint`?
- [ ] ~~tried existing tests with `sudo port test`?~~ (No tests to perform)
- [x] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?

### CODE REVIEW

File: audio/abcde/Portfile [hunk 1]
```diff
@@ -1,9 +1,11 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem              1.0
+PortGroup               perl5 1.0
 
 name                    abcde
-version                 2.8.1
+version                 2.9.1
+revision                1
```
Comment (pmetzger) [inline audio/abcde/Portfile]: This shouldn't be `revision 1` since you're bumping the `version` in this commit

Comment (chrstphrchvz) [reply audio/abcde/Portfile]: This is currently two separate commits: ffeb904 bumps version to 2.9.1, 24c8338 bumps revision to 1. (GitHub's "Files changed" tab shows all commits together as if they were one commit.)

Comment (pmetzger) [reply audio/abcde/Portfile]: So what? They'll be merged as a group, so there won't be some point where 2.9.1 is available to users without the rest. (And in fact, you should probably squash these...)

**Comment by pmetzger** [comment]
LGTM.

**Comment by pmetzger** [comment]
Merged. Thank you, @chrstphrchvz!

File: audio/abcde/Portfile [hunk 2]
```diff
@@ -19,25 +20,29 @@ long_description        abcde is a frontend command-line utility (actually, a \
...
                         port:lame \
                         port:flac \
                         port:cd-discid \
                         port:cdparanoia \
                         port:id3v2 \
-                        port:normalize
+                        port:normalize \
+                        port:p5-digest-sha \
+                        port:p5-musicbrainz-discid \
+                        port:p5-webservice-musicbrainz
```
Comment (l2dy) [inline audio/abcde/Portfile]: These `p5-*` ports aren't frozen at 5.26, so you should use `p${perl5.major}-` instead. See port `mosh` for an example.

Comment (chrstphrchvz) [reply audio/abcde/Portfile]: I have opened #1982 to address this.

**Pull request merged**